apps/confidential/confidential.go

```diff
@@ -381,12 +381,12 @@ func (cca Client) AcquireTokenOnBehalfOf(ctx context.Context, userAssertion stri
 }
 
 // Account gets the account in the token cache with the specified homeAccountID.
-func (cca Client) Account(homeAccountID string) Account {
-	return cca.base.Account(homeAccountID)
+func (cca Client) Account(ctx context.Context, homeAccountID string) Account {
```

Comment:
@chlowell @element-of-surprise - is there a way in Go to add the context param as suggested without a breaking change? Varargs + interface maybe (http://changelog.ca/log/2015/01/30/golang) ?

Reply:
Yes, however it would be bizarre to take a `Context` as anything but the first argument. Google established that convention in the standard library and chose to add duplicate `*Context` or `*WithContext` methods where needed ([for example](https://pkg.go.dev/net/http#NewRequest)).

Reply:
If we want to support the change, I suggest the superset as I mention in the thread below (I could assign this to myself and make it happen).

Since we don't treat this as a preview, we shouldn't make a breaking change.  For us, a general release means that we support a very specific set of features in MSAL.  

I think for us to consider doing a breaking release, we should have a laundry list of things we want to change or better support.    Also, we should probably be fairly close to having features that let us make a general release.  That way we at least know we won't need/want to make future changes to support the feature set.

Reply:
@bgavrilMS -- You want me to take this and make the changes?

Reply:
Go right ahead @element-of-surprise